Lib/wavm-c/wavm-c.cpp

```diff
@@ -876,6 +876,26 @@ size_t wasm_func_result_arity(const wasm_func_t* function)
 {
 	return getFunctionType(function).results().size();
 }
+bool wasm_func_name(const wasm_func_t* function, char* out_message, size_t* inout_num_message_bytes)
+{
+	if (!function || !function->mutableData) {
+		return false;
+	}
```

Comment:
Can you reformat this to make it pass the clang-format CI check?
https://dev.azure.com/WAVM/WAVM/_build/results?buildId=976&view=logs&jobId=912ee087-e926-5f03-d629-076295d314d0&taskId=dad19407-d860-540f-9abb-2150b15248df&lineStart=20&lineEnd=21&colStart=1&colEnd=1